src/openfermion/transforms/opconversions/commutator_diagonal_coulomb_operator.py

```diff
@@ -14,8 +14,9 @@
 import warnings
 
 from openfermion.ops.operators import FermionOperator
-import openfermion.transforms.opconversions.term_reordering\
-    as term_reordering
+from openfermion.transforms.opconversions import term_reordering
+# import openfermion.transforms.opconversions.term_reordering\
+#     as term_reordering
```

Comment:
Not sure why this is commented out, maybe we can just delete it? (No idea if this was from me or you @ncrubin ).

Reply:
That was me.  I was sorting out what was causing the issue and I changed this to a from import.  I'll fix it.

Reply:
Ah ok. Did that make a difference? I feel a bit lost in this damn mess of imports we have.

Reply:
The problem wasn't the importing structure.  A file that got moved `double_commuting...two_body` was being incorrectly imported (wrong location).  Thus tests were failing.  The error message was different from when you get circular imports...though sounded similar.  

All that I did to fix was to import from the correct location.

Reply:
Hah, very sorry about that then. For some reason tests were passing on my computer... Or at least I thought they were. (I mean, this isn't the kind of error that should pass on any machine)